js/lib/beautify.js

```diff
@@ -423,7 +448,11 @@
                 return
             }
 
-            if ((opt.preserve_newlines && current_token.wanted_newline) || force_linewrap) {
+            var shouldPreserveOperatorNewline = (opt.operator_position === OPERATOR_POSITION.preserve_newline) &&
+                in_array(flags.last_text, OP_POSITION_OPERATORS) ||
+                in_array(current_token.text, OP_POSITION_OPERATORS);
+                
+            if (((opt.preserve_newlines || shouldPreserveOperatorNewline) && current_token.wanted_newline) || force_linewrap) {
```

Comment:
I will wait to hear our resolution surrounding default behavior before discussing your above comments.